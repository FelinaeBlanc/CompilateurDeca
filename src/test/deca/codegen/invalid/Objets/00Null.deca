class A {
    int a=0;
}

{
    A a = null;
    int x = a.a;
}
