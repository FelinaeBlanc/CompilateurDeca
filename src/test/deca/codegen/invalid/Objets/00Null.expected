  ** IMA ** ERREUR ** Ligne 23 : 
    Adressage indirect : la base ne contient pas une adresse
